Sprint-1/interpret/to-pounds.js

```diff
@@ -21,7 +21,49 @@ console.log(`£${pounds}.${pence}`);
 // The program then builds up a string representing the price in pounds
 
 // You need to do a step-by-step breakdown of each line in this program
+/*
+const penceString = "399p"; - Declares a constant variable penceString and assigns it the string value "399p"
+const penceStringWithoutTrailingP = penceString.substring(0,penceString.length - 1); - Uses the substring() method to extract a portion of the penceString
+0: Starts from the beginning of the string.
+penceString.length - 1: Ends one character before the end of the string, effectively removing the trailing "p".
+
+const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+Uses the padStart() method to add leading zeros to the penceStringWithoutTrailingP.
+3: Specifies the desired total length of the string after padding.
+"0": The character to use for padding.
+This ensures that the number has at least 3 digits
+
+const pounds = paddedPenceNumberString.substring(0,paddedPenceNumberString.length - 2);
+Extracts the pounds part from the padded string.
+0: Starts from the beginning of the padded string.
+paddedPenceNumberString.length - 2: Ends two characters before the end, capturing the pounds part.
+
+const pence = paddedPenceNumberString.substring(paddedPenceNumberString.length - 2).padEnd(2, "0");
```

Comment:
Can the code still function as before if we removed `.padEnd(2, "0")` from this code?